<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDirectoryPickerWidgetClass</class>
 <widget class="QWidget" name="QDirectoryPickerWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDirectoryPickerWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="lineDirectory">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonOpenDirectory">
     <property name="toolTip">
      <string>Opens the currently selected directory</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSelectDirectory">
     <property name="toolTip">
      <string>Opens a dialog to select a new directory</string>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::FolderNew"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
